package.json

```diff
@@ -9,7 +9,7 @@
   },
   "main": "src/index.js",
   "engines": {
-    "node": ">=14"
+    "node": ">=16.14"
```

Comment:
Same as in `docusaurus`: https://github.com/facebook/docusaurus/blob/f5ae537d3ec6b2745957b14ead890187c70b302f/packages/docusaurus/package.json#L128